packages/portal-marketmaker/src/index.js

```diff
@@ -0,0 +1,407 @@
+
+const provider = new ethers.providers.JsonRpcProvider(ethRpcProvider);
+const signer = new ethers.Wallet(privateKey, provider);
+
+const portal = new ethers.Contract(portalAddress, portalJSONABI, provider);
+const portalWithSigner = portal.connect(signer);
+
+const wbtc = new ethers.Contract(wbtcAddress, wbtcAbi, provider);
+const wbtcWithSigner = wbtc.connect(signer);
+
+
+var minblock = 0;
+
+function handlePendingOrder(order) {
```

Comment:
i'd recommend giving these functions one-liners too

for this one, if i'm understanding it right, something like

```
// Adds order we just placed to our liquidity totals [bid/ask][Sats/Liquidity] 
```